.ci/ci.yaml

```
@@ -36,7 +36,7 @@ jobs:
 - job: macOS
   timeoutInMinutes: 360
   pool:
-    vmImage: macOS-latest
+    vmImage: macOS-10.15
```

```suggestion
    vmImage: macOS-latest
```

macOS latest should be fine.